ci/bin/build-and-test

```diff
@@ -7,7 +7,19 @@ template_path = File.join(root_path, "template.rb")
 builds_path = File.join(root_path, "tmp/builds")
 app_name = ENV.fetch("APP_NAME", "app")
 app_path = File.join(builds_path, app_name)
-config_path = File.join(root_path, ENV.fetch("CONFIG_PATH", "ci/configs/basic.yml"))
+
+config_path = if ENV.fetch("CONFIG_PATH", "") == ""
+                FileUtils.cp(
+                  File.join(root_path, "ackama_rails_template.config.yml"),
+                  File.join(builds_path, "ackama_rails_template.config.yml")
+                )
+
+                nil
```

Comment:
Having two copies of the `ackama_rails_template.config.yml` might be a _bit_ confusing for devs working on this project. Could we avoid this by setting `config_path` to the top level file if no `CONFIG_PATH` env var provided?

```suggestion
                File.join(root_path, "ackama_rails_template.config.yml"),
```